### ISSUE TITLE
[Bug Report] Radio Group hijacks events of other input control

### ISSUE THREAD

**Issue by mrin9** [ElemeFE/element#11906]
### Element UI version
2.4.3

### OS/Browsers version
all

### Vue version
2.5.16

### Reproduction Link
[https://jsfiddle.net/mmx38qxw/3343/](https://jsfiddle.net/mmx38qxw/3343/)

### Steps to reproduce
Created a fiddle (https://jsfiddle.net/mmx38qxw/3343/) with a radio group. 
and there is a text-input inside it.
When I am in the text input and press left or right arrow keys, 
it is being listened by the radio-group. and in the process fails and generates console errors

### What is Expected?
radio group should listen to only its own events

### What is actually happening?
Radio group is listening to the arrow-keydown events of input

**Comment by jikkai** [comment]
You can stop event propagation with `@keydown.stop`
https://jsfiddle.net/mmx38qxw/3345/

**Comment by mrin9** [comment]
can you please reopen the issue, the solution you gave is just an work-around. 
I do like to propagate the events to parent, 
Plus stop propagation WONT WORK if i use ```<el-input>``` instead of an ```<input>```

**Comment by jikkai** [comment]
Sorry Bro, it's my fault. I created a PR to fix this issue.

BTW, if you want to use `el-input` instead of native `input`, you can still make it work with `@keydown.native.stop`. Anyway, thx for your suggestion :)

**Comment by mrin9** [comment]
thanks for reopening ... you guys are doing a great job and I appreciate the time and effort you are putting into this
